app/src/main/java/com/example/ai_stuttering_assistant/MainActivity.kt

```diff
@@ -65,8 +71,22 @@ class MainActivity : ComponentActivity() {
         Log.d("APIKey", apiKey)
         virtualAssistant = VirtualAssistant(apiKey)
         mediaPlayer = MediaPlayer()
+
+        enableEdgeToEdge()
+
+        if( android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.P ) {
+            window.attributes.layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES
+        }
+
         setContent {
-            Interface(this)
+            AIStutteringAssistantTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    Interface( this )
+                }
+            }
```

Comment:
The `AIStutteringAssistantTheme` calls the theming details in `Theme.kt` and the `Surface` grabs the background colour from the `MaterialTheme` which again comes from `Theme.kt`. Within both of these, the `Interface` is called and any new content should be added here or within the `Interface` composable function itself.